tests/compute/texture-subscript-uint-cuda.slang

```diff
@@ -0,0 +1,92 @@
+// Test for verifying subscript operator support with uint types in cuda.
+
+//TEST(compute):COMPARE_COMPUTE_EX:-cuda -compute -shaderobj
```

Comment:
Just FYI, I think that the `-shaderobj` flag doesn't actually do anything these days